exercises/for_the_sake_of_argument/problem.md

```diff
@@ -19,12 +19,12 @@ Feature #2 is a C++ change. To implement it, you'll need to learn about argument
 
 ### I wanted an argument!
 
-When you use the macro `NAN_METHOD()`, you automatically have access to an `args` array inside the method even though you don't see it declared. The elements of this array correspond to the arguments passed in, so `args[0]` is the first argument.
+When you use the macro `NAN_METHOD()`, you automatically have access to an `info` array inside the method even though you don't see it declared. The elements of this array correspond to the arguments passed in, so `info[0]` is the first argument.
 
 Each element in the array is an object is a V8 type with a special function `As<Type>()` allowing you to *cast* it to a different V8 type. e.g. to use a `v8::Number` type as the 3rd argument in your method, write:
```

Comment:
Hmm, I guess if we change everything to use `Nan::To<>` then we need a paragraph or two explaining how `Nan::Maybe` and `MaybeLocal` stuff works.